app/assets/stylesheets/screen.scss

```diff
@@ -692,17 +647,19 @@ div#keycut-help {
   .list-group-item {
     border-radius: 0;
     border-width: 1px 0 0 0;
-  }
 
-  > .list-group-item:first-child {
-    border-top-width: 0;
+    > &:first-child {
```

Comment:
I don't think the `>` is necessary here. The first child element will be a direct descendant anyway.